internal/test/wsu/wsu_test.go

```diff
@@ -205,6 +238,27 @@ func testWorkerLabelsArePresent(t *testing.T) {
 		"expected worker label to be present on the Windows node but did not find any")
 }
 
+// doesFileContain checks if the given file on the remote Windows virtual machine contains given string
+func doesFileContainMatchingString(fileName, matchString string) (bool, error) {
```

Comment:
It would be nice if the file is read just once and not every time this function is called